src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -89,13 +89,14 @@
                         if (!(ex is ExitApplicationException<ExitReason>))
                         {
                             // These exceptions escaped our net and must be logged here
-                            RuntimeErrors |= Errors.LogUnhandledEngineException(_rootContext, ex);
+                            Errors.LogUnhandledEngineException(_rootContext, ex);
```

Comment:
>Errors.LogUnhandledEngineException [](http://example.com/codeflow?start=28&length=34)

so this call may throw exception, should we add a 'catch' block in the first 'try' block to catch it?